Install-Guide/Community_Packages.md

```diff
@@ -1,35 +1,38 @@
 ### Community Packages
 
-An **X** means packages are planned to be in the respective repository.  
-A **—** means we have no plans to build the version for the repository.  
-**d.g.o** means packages will (also) be provided on https://download.gluster.org  
-**DNF** means the packages are installed from the Fedora Updates or Updates-testing repos.  
+An **X** means packages are provided in the respective repository.  
```

Comment:
Sorry, my bad. It wasn't a list. It looks just fine on http://dillinger.io/ and Github seems to [misrender](https://github.com/kalebskeithley/glusterdocs/blob/715fe3856d0dcf04dc191625ab1e5eaf6ca47969/Install-Guide/Community_Packages.md) it. You just have to add two spaces at the end of each intended linebreak for it to be rendered as new line.

I wanted to fix this for you and merge but unlike gerrit, github doesn't allow adding commits/amends to someone else's PR.